README.md

```diff
@@ -420,6 +414,100 @@ enable this feature, just create a new organization with the
 
 `Are you using OpenWISP Subscriptions to provide paid subscriptions for WiFi plans or identity verification via credit/debit card?`
 
+### Translations
+
+Translations are loaded at runtime from the JSON files that were compiled during
+the build process according to the available languages defined and taking into
+account any customization of the translations (more on [defining-available-languages](#defining-available-languages),
+[add translations](#add-translations) and [customizing translations](#customizing-translations-for-a-specific-language)).
+
+#### Defining available languages
+
+If there is more than one language in `i18n/` directory then update the organization
+configuration file by adding the support for that language like this:
+
+```yaml
+languages:
+  - text: "English"
+    slug: "en"
+  - text: "Italian"
+    slug: "it"
+```
+
+#### Add translations
+
+Translation file with content headers can be created by running:
+
+`yarn translations-add {language_code} i18n/{file_name}.po`
+
+Here `file_name` can be `{orgSlug}_{language_code}.custom.po`, `{language_code}.custom.po\` or
+`{language_code}.po`.
+
+See [update-translations](#update-translations) for extracting translation placeholders
+from the code.
+
+#### Update translations
+
+For extraction or updating translations in the `.po` file, Use the command:
+
+`POFILE=<path-to-po-file> yarn translations-update`
```

Comment:
Because the first argument needs to be the path to po file and the second argument needs to be src files.

https://github.com/ttag-org/ttag-cli/blob/76b340e87b550193da03b9a3f88dbe88b648fc93/src/index.ts#L226-L236